azure_jumpstart_ag/artifacts/PowerShell/Modules/manufacturing.psm1

```diff
@@ -293,7 +293,8 @@ function Deploy-AIO {
         Write-Host "AIO deployed successfully on the $clusterName cluster" -ForegroundColor Green
         Write-Host "`n"
         Write-Host "[$(Get-Date -Format t)] INFO: Started Event Grid role assignment process" -ForegroundColor DarkGray
-        $extensionPrincipalId = (az k8s-extension show --cluster-name $arcClusterName --name "mq" --resource-group $resourceGroup --cluster-type "connectedClusters" --output json | ConvertFrom-Json).identity.principalId
+        $extName = "mq-" + $using:namingGuid
+        $extensionPrincipalId = (az k8s-extension show --cluster-name $arcClusterName --name $extName --resource-group $resourceGroup --cluster-type "connectedClusters" --output json | ConvertFrom-Json).identity.principalId
```

Comment:
It seems the guid added is random and is not the namingGuid we have. It seems a new change has been introduced to the extensions naming. I think we can better filter by type 

```suggestion
        $extensionPrincipalId =(az k8s-extension list --cluster-name $arcClusterName --resource-group $resourceGroup --cluster-type "connectedClusters" --query "[?extensionType=='microsoft.iotoperations.mq']" --output json | ConvertFrom-Json)[0].identity.principalId
```